### PR TITLE
Public release

### PR DESCRIPTION
**Pull request by ryexley**
Merging `develop` into `main` to build and deploy publicly.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: theartofeducation.github.io
```
Comment (theantichris) [inline .github/workflows/main.yml:1]: We need to rename this file or the main workflow on the template-repo. They are both named main.

The one that comes in from the template will contain workflows that every project will use, like the Markdown linter. This one is specific to the project.

Thoughts?

File: src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { FormattedMessage, useIntl } from "gatsby-plugin-intl"
+// import { Link } from "gatsby"
```
Comment (theantichris) [inline src/pages/index.js:3]: 🤮

**Review by theantichris — CHANGES_REQUESTED** [review]
There are some files that say they can be deleted if they aren't used. I'm not sure if they will be used or not so just mentioning.

File: src/pages/index.js (hunk 2 above)
Comment (ryexley) [reply src/pages/index.js:3]: Aw crap, yeah ... missed that. I can fix that.

File: .github/workflows/main.yml (hunk 1 above)
Comment (Thomas-Jean) [reply .github/workflows/main.yml:1]: In my mind, main is the task the runs on every push, like linting and testing. In JS we would test with jest, but with python we would test with pytest. Each repo will need specific tasks for each kind of project.

Comment (theantichris) [reply .github/workflows/main.yml:1]: We decided on manual intervention for that.

**Review by theantichris — APPROVED** [review]

**Pull request merged**